descriptors/ModuleDescriptor-template.json

```diff
@@ -137,6 +137,10 @@
     {
       "id": "permissions",
       "version": "5.7"
+    },
+    {
+      "id": "consortia",
```

Comment:
is consortia available for non-ecs?

Reply:
hi @NikitaSedyx , the consortia logic is used as in mod-bulk-operations module. But the last one does not have consortia interface in module descriptor. May be it is better to use optional or remove it. Let's double check.

Reply:
it must be optional, consortia is not part of platform - you had similar issue before, time to think what you include in required interfaces

Reply:
additionally please create a PR to add interfaces in bulk ops

Reply:
Updated